Sources/MapboxNavigation/WayNameView.swift

```diff
@@ -67,7 +67,7 @@ open class WayNameView: UIView {
     ///
     /// This method attempts to extract the road name and shield image as well as styling information and tries to display it. Return result shows if it was a success.
     @discardableResult
-    public func setupWith(roadFeature feature: MBXFeature, using style: MapboxMaps.Style?) -> Bool {
+    public func setupWith(roadFeature feature: MapboxCommon.Feature, using style: MapboxMaps.Style?) -> Bool {
```

Comment:
This is a public method, but `MapboxCommon.Feature` is an Objective-C type rather than a proper Swift type. Rename this method to `update(feature:style:)` and change the first argument’s type to `Turf.Feature`.

/ref https://github.com/mapbox/mapbox-navigation-ios/pull/3019#discussion_r641713731

Reply:
I've changed method signature, on the other hand I think we still need to use `MapboxCommon.Feature` instead of Turf.Feature, because current method is called from `MapboxMap.queryRenderedFeatures(at:options:completion:)`, which provides `MapboxCommon.Feature` as a result. I'm not sure why this method is public though, I think we can make it internal.

Reply:
We should convert `MapboxCommon.Feature`s to `Turf.Feature`s.